### ISSUE TITLE
[Question] how to set Focal loss Parameters

### ISSUE THREAD

**Issue by vkherde** [mvoelk/ssd_detectors#14]
Hi @mvoelk 

I am trying to use focal loss in SSD model, as per the code in DSOD_train.ipynb, lines
 -  loss = SSDFocalLoss(lambda_conf=10000, class_weights=class_weights)
 -  class_weights = np.array([0.00007205, 1.3919328 , 1.43665262, 1.30902077, 1.36668928, 1.2391509 , 1.21337629, 0.41527107, 1.1458096 , 0.29150119, 1.25713104, 0.61941517, 1.03175604, 1.21542005, 1.01947561, 0.0542007 , 1.12664538, 1.14966073, 1.12464889, 1.49998021, 1.09218961])

I want to understand how these parameters were calculated for "lamba_conf" and "class_weights" please confirm if you got a chance. Many thanks indeed

Best Regards 
Vaibhav

**Comment by mvoelk** [comment]
I did not spend much time on the DSOD model.

The value for lambda is probably much too high. I took this value only for some experiments. A value of `100.0`, as I used it for the SegLink models, should work find. I will also change that in the code.

The `class_weight` values were calculated using `compute_class_weights` method in the `PriorUtility`.
https://github.com/mvoelk/ssd_detectors/blob/3ef2c31dd6486a18940ec79f448f4f2e7d1c9df8/ssd_utils.py#L552-L561
Roughly speaking, they are the normalized inverse of the class frequencies in the local ground truth.
